cabal-install/Distribution/Client/CmdTest.hs

```diff
@@ -206,9 +210,25 @@ data TargetProblem =
    | TargetProblemIsSubComponent   PackageId ComponentName SubComponentTarget
   deriving (Eq, Show)
 
-reportTargetProblems :: Verbosity -> [TargetProblem] -> IO a
-reportTargetProblems verbosity =
-    die' verbosity . unlines . map renderTargetProblem
+reportTargetProblems :: Flag Bool -> Verbosity -> [TargetProblem] -> IO a
```

Comment:
`Verbosity` normally always comes first unless there are special reasons.

Reply:
Good point, I'll send a follow-up to clean that up.